packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.styles.ts

```diff
@@ -46,17 +46,19 @@ const useStyles = makeStyles({
     color: tokens.colorNeutralForeground2,
   },
   calloutBlockContainer: {
-    fontSize: 'fontSizeBase200',
+    fontSize: tokens.fontSizeBase200,
     marginTop: '13px',
     color: tokens.colorNeutralForeground2,
+    borderLeft: 'var(--lineColor)',
```

Comment:
what value does lineColor represent

Reply:
the value is provided in the popover class as `4px solid ${color}`,